activerecord/test/fixtures/subscribers.yml

```diff
@@ -6,6 +6,6 @@ second:
   nick: webster132
   name: David Heinemeier Hansson
 
-thrid:
+third:
```

Comment:
If nobody is using the fixture I think we could remove it. 
Didn't look much into it to be honest. But I am always for removing not used code.